src/Model/Resolver/WishlistItemsResolver.php

```diff
@@ -221,12 +219,17 @@ public function resolve(
             $buyRequestOption = $wishlistItem->getOptionByCode('info_buyRequest');
             $options = $this->getItemOptions($wishlistItem, $type);
 
+            $product = $wishlistItem->getProduct();
+
+            $productPriceIncTax = $product->getPriceInfo()->getPrice('final_price')->getMinimalPrice()->getValue();
+            $productPriceExcTax = $product->getPriceInfo()->getPrice('final_price')->getMinimalPrice()->getValue('tax');
+
             $data[] = [
                 'id' => $wishlistItemId,
                 'qty' => $qty,
                 'sku' => $this->getWishListItemSku($wishlistItem),
-                'price' => $price,
-                'price_without_tax' => $priceWithoutTax,
+                'price' => $productPriceIncTax,
+                'price_without_tax' => $productPriceExcTax,
```

Comment:
Since you are not using $price and $priceWithoutTax, please clean the code and remove unused variables and methods.